Week1/node.js

```diff
@@ -0,0 +1,184 @@
+];
+
+for (let i in insert_cities) {
+  // console.log('Going to run ', insert_cities[i]);
+  database.query(insert_cities[i], error => {
+    if (error) {
+      throw error;
+    }
+    console.log([i] + 'success');
+  });
+}
+
+// What are the names of the countries with population greater than 8 million
+database.query(`SELECT * FROM country WHERE Population > 8000000`, (error, result) => {
+  if (error) throw error;
+
+  console.log(`\n Countries with population more than 8 million:`);
+  result.forEach(country => {
+    console.log(`${country.Name} , ${country.Population}`);
+  });
+});
+
+// Names of the countries that have “land” in their names ?
+database.query(`SELECT * FROM country WHERE Name LIKE '%land%' `, (error, result) => {
+  if (error) throw error;
+  console.log(`\n Countries that have “land” in their names:`);
+  result.forEach(land => {
+    console.log(land.Name);
+  });
+});
+
+// Cities with population between 500,000 and 1 million ?
+database.query(
+  `SELECT * FROM city WHERE Population BETWEEN 500000 AND 1000000 `,
+  (error, result) => {
+    if (error) throw error;
+    console.log(`\n Cities with population between 500,000 and 1 million is :`);
+    result.forEach(city => {
+      console.log(city.Name);
+    });
+  },
+);
+
+// Countries in the Europe continent ?
+
+database.query(`SELECT * FROM country WHERE Continent = 'Europe' `, (error, result) => {
+  if (error) throw error;
+
+  console.log(`\n Countries in Europe continent:`);
+  result.forEach(country => {
+    console.log(country.Name);
+  });
+});
+
+// Countries in descending order, based on their surface areas.
+
+database.query(`SELECT * FROM country ORDER BY SurfaceArea DESC`, (error, result) => {
+  if (error) throw error;
+
+  console.log(`\n Countries in descending order based on their surface areas:`);
+  result.forEach(country => {
+    console.log(`${country.Name} => ${country.SurfaceArea}`);
+  });
+});
+
+//cities in the Netherlands?
+database.query('SELECT * FROM city WHERE country_code = "NLD"'),
+  (error, result) => {
+    if (error) throw error;
+
+    console.log(`\n The cities in the Netherlands are :`);
+    result.forEach(city => {
+      console.log(city.name);
+    });
+  };
+
+//==========================
+// What's the population of Rotterdam?
+database.query('SELECT * FROM city WHERE name = "Rotterdam"', (error, result) => {
+  if (error) throw error;
+
+  console.log(` Population of Rotterdam is:`);
+  result.forEach(city => {
+    console.log(city.Population);
+  });
+});
+
+// top 10 countries based on surface area?
+database.query('SELECT * FROM country ORDER BY surface_area DESC LIMIT 10', (error, result) => {
+  if (error) throw error;
+
+  console.log(`\n Top 10 countries based on surface area:`);
+  result.forEach(country => {
+    console.log(country.Name);
+  });
+});
+
+// top 10 cities with the highest population?
+database.query('SELECT FROM city ORDER BY population DESC LIMIT 10', (error, result) => {
```

Comment:
select what??

Reply:
Thanks for mention that , it is corrected now